drivers/scheduler/rke/rke.go

```diff
@@ -270,6 +272,53 @@ func (r *Rancher) SaveSchedulerLogsToFile(n node.Node, location string) error {
 	return err
 }
 
+// RemoveNamespaceFromProject moves namespace to no project
+func (r *Rancher) RemoveNamespaceFromProject(nsList []string) error {
+	for _, ns := range nsList {
+		ns, err := core.Instance().GetNamespace(ns)
+		if err != nil {
+			return err
+		}
+		delete(ns.Labels, "field.cattle.io/projectId")
```

Comment:
Would it be a good idea to use a loop here?  where you just fetch ns.labels and ns.annotatios and keep just one delete call inside it?

Reply:
done